CMakeLists.txt

```diff
@@ -114,7 +114,6 @@ else()
   set(_default_examples_unit_tests OFF)
 endif()
 option(ACTS_BUILD_EXAMPLES_UNITTESTS "Build unit tests" ${_default_examples_unit_tests}) # default: ACTS_BUILD_UNITTESTS AND ACTS_BUILD_EXAMPLES
-option(ACTS_BUILD_NONCOMPILE_TESTS "Build tests that check build failure invariants" OFF)
```

Comment:
unrelated?

Reply:
Yeah that's a leftover from a previous PR. I can break it out if you prefer, but the option is already unused now so it seemed like an easy fix on the side.

Reply:
Hmm just mention it in the commit message then maybe.